src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -3913,12 +3947,22 @@ class C
                 Await state.WaitForAsynchronousOperationsAsync()
 
                 Assert.Equal("#", state.GetLineFromCurrentCaretPosition().GetText())
-                Await state.AssertNoCompletionSession()
+                If completionImplementation = CompletionImplementation.Legacy Then
+                    Await state.AssertNoCompletionSession()
+                Else
+                    Await state.AssertCompletionSession() ' Agreed on changing the behavior
+                End If
+
                 state.SendTypeChars("reg")
                 Await state.AssertSelectedCompletionItem(displayText:="region")
                 state.SendReturn()
                 Await state.AssertNoCompletionSession()
-                Assert.Equal("    #region", state.GetLineFromCurrentCaretPosition().GetText())
+                If completionImplementation = CompletionImplementation.Legacy Then
+                    Assert.Equal("    #region", state.GetLineFromCurrentCaretPosition().GetText())
+                Else
+                    Assert.Equal("#region", state.GetLineFromCurrentCaretPosition().GetText())
```

Comment:
why are these different?

Reply:
The Editor does not provide us capabilities to test Up and Down key presses in the completion popup. We opened an issue on them to implement this and similar tests on their side.

---
In reply to: [222523246](https://github.com/dotnet/roslyn/pull/29016#discussion_r222523246) [](ancestors = 222523246)

Reply:
Resolved the issue with missed spaces before #region or #endregion